napari/resources/_icons.py

```diff
@@ -154,7 +155,7 @@ def write_colorized_svgs(
 
 
 def _theme_path(theme_name: str) -> Path:
-    return ICON_PATH / '_themes' / theme_name
+    return Path(user_cache_dir()) / '_themes' / theme_name
```

Comment:
Does this mean now that multiple napari versions (e.g. from different conda environments) will share the same theme directory? That might be a problem? Should there perhaps be a hash of the version or of the executable path there?

Reply:
This is the same problem as with settings.

Reply:
I change to cache dir be separate per version

Reply:
Yeah but settings has explicit safeguards (?) for incorrect or deprecated keys, where I don't think the theme does? Anyway thanks for the update! 🙏